org.jacoco.build/pom.xml

```diff
@@ -265,6 +265,7 @@
         <filtering>false</filtering>
         <excludes>
           <exclude>**/*.java</exclude>
+          <exclude>**/*.kt</exclude>
```

Comment:
I wonder whether Kotlin specific configurations should really go into our global parent pom (thus theoretically also affecting also our productive modules). If we keep adding integration tests for other languages it might become confusing here. I personally would prefer to move Kotling specific config to `org.jacoco.core.test.validation.kotlin`. Sure, probably need an new property.